swift-sdk/Internal/InAppHelper.swift

```diff
@@ -39,19 +39,22 @@ struct InAppHelper {
                 return nil
             }
             return .localResource(name: urlPath)
-        case .itbl:
-            return .iterableCustomAction(name: dropScheme(urlString: url.absoluteString, scheme: UrlScheme.itbl.rawValue))
+        case .iterable:
+            return .iterableCustomAction(name: dropScheme(urlString: url.absoluteString, scheme: scheme.rawValue))
         case .action:
-            return .customAction(name: dropScheme(urlString: url.absoluteString, scheme: UrlScheme.action.rawValue))
+            return .customAction(name: dropScheme(urlString: url.absoluteString, scheme: scheme.rawValue))
+        case .backwardCompat:
+            return .customAction(name: dropScheme(urlString: url.absoluteString, scheme: scheme.rawValue))
         case .other:
             return .regularUrl(url)
         }
     }
     
     private enum UrlScheme : String {
         case applewebdata = "applewebdata"
-        case itbl = "itbl"
+        case iterable = "iterable"
         case action = "action"
+        case backwardCompat = "itbl"
```

Comment:
I think it might be a bit clearer to name this `itbl` and add a comment that it's for backwards compatibility since all the other values are following the convention of using the scheme name itself.